provider/.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch Provider",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "cmd/pulumi-resource-docker/main.go",
+            "cwd": "/Users/tkappler/pulumi/pulumi-awsx/examples/ts-ecr-simple"
```

Comment:
This does not look like a good `"cwd"` to check in. 

In general, we don't check in `.vscode` project files to our providers.